src/main/java/io/cdap/e2e/pages/locators/CdfBigQueryPropertiesLocators.java

```diff
@@ -124,4 +124,9 @@ public static WebElement partitioningType(String option) {
 
   @FindBy(how = How.XPATH, using = "//*[@data-cy='dedupeBy']//*[@data-cy='value']")
   public static WebElement dedupeFunctionName;
+
+  public static WebElement locatePropertyInput(String pluginProperty) {
```

Comment:
1. The field has a + button which allows the field to take a dynamic list of inputs
2. Locator and the corresponding action needs to handle this
3. Move the locator and the action to the CdfPluginProperties... class
4. Ref: Salesforce

Reply:
Resolved.